extensions/AzureAISearch/AzureAISearch/AzureAISearchFiltering.cs

```diff
@@ -13,22 +13,22 @@ internal static class AzureAISearchFiltering
     /// <summary>
     /// Build a search query optimized to scale for in case a key has several (hundreds+) values
     /// Note:
-    /// * One filter can have multiple key-values: these are combined with AND conditions
-    /// * Multiple filters are combined using OR
-    /// * Multiple filters can apply to the same key:
-    ///     * if filters have only one condition, these are rendered using "search.in"
-    ///     * if filters have multiple conditions, the values are combined with OR
+    /// * One Filter can have multiple key-values: these are combined with AND conditions
+    /// * Multiple Filters are combined using OR
+    /// * Multiple Filters can apply to the same key:
+    ///     * if Filters have only one condition, these are rendered using "search.in"
+    ///     * if Filters have multiple conditions, the values are combined with OR
```

Comment:
looks like some find&replace went wild here :-) could you undo all the "filter" to "Filter" changes?

Reply:
fixed